pkg/pagerduty/pagerduty.go

```diff
@@ -126,6 +126,38 @@ func (p PagerDuty) AddNote(incidentID string, noteContent string) error {
 	return nil
 }
 
+// GetAlerts will retrieve the alerts for a specific incident
+func (p PagerDuty) GetAlerts(incidentID string) ([]Alert, error) {
+	ctx, cancel := context.WithTimeout(context.Background(), pagerDutyTimeout)
+	defer cancel()
+
+	o := sdk.ListIncidentAlertsOptions{}
+
+	alerts, err := p.c.ListIncidentAlertsWithContext(ctx, incidentID, o)
+	if err != nil {
+		sdkErr := sdk.APIError{}
+		if errors.As(err, &sdkErr) {
```

Comment:
To get rid of one level of nestign, write the following here:

```
if !error.As(err, &sdkErr){
 return nil, UnknownGetAlertsError{Err: err}
}
```

Reply:
this is probably something that you can also put into commonErrHandler? Not sure right now

Reply:
did something similar, resolving this and please reopen a different comment on the new structure if it's still not good